doc/ref/grplib.xml

```diff
@@ -99,6 +99,31 @@ calculations.
 <#Include Label="SuzukiGroup">
 <#Include Label="ReeGroup">
 
+<Subsection Label="Generator Names">
+<Heading>Generator Names</Heading>
+For groups created as finitely presented groups of polycyclic groups, the
```

Comment:
Not sure I understand... perhaps this was meant?
```suggestion
For groups created as finitely presented groups or polycyclic groups, the
```

Optional remark: This might then be slightly confusing to *some* people: mathematically, of course polycyclic groups are also finitely presented groups. Perhaps `FpGroups` vs `PcGroups` is meant here? Of course that's a bit technical. Perhaps like this:
```suggestion
For groups created as finitely presented groups, including polycyclic groups, the
```